packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5422,8 +5426,13 @@ abstract class RenderObjectElement extends Element {
   RenderObjectWidget get widget => super.widget as RenderObjectWidget;
 
   /// The underlying [RenderObject] for this element.
+  ///
+  /// If this element has been [unmount]ed, this getter will throw.
   @override
-  RenderObject get renderObject => _renderObject!;
+  RenderObject get renderObject {
+    assert(_renderObject != null, 'Object disposed');
```

Comment:
Object  disposed --> `RenderObject disposed?` or `'$runtimeType unmounted'`?

Reply:
Going with `$runtimeType` disposed`, since ROs don't have a notion of unmounting.

Reply:
But isn't this the case where this RenderObjectElement has been unmounted not really the case that the RenderObject has been disposed? The RenderObject probably has been disposed given it is null.

Reply:
Ahh I missed which thing I was in. I see what you're saying now. Will change to unmounted.